ui/src/app/projects/projects.component.html

```diff
@@ -4,7 +4,7 @@
       <input matInput
              autofocus
              class="input-field"
-             placeholder="Project ID"
+             placeholder="ProjectID"
```

Comment:
Why the change? I think it looks cleaner with the space

Reply:
No good reason – happened during debugging the tests; fixed.